src/containers/Messages/MessagesList.js

```diff
@@ -30,15 +34,34 @@ const getContacts = async (setContacts) => {
   );
 };
 
-const Messages = () => {
+const addAdminToConversation = async (history, user, room) => {
+  await database.ref(`messages/${room}`).update({ admin: user.uid });
+  history.push(`/messages/${room}`);
+};
```

Comment:
Why need to add admin to conversation?

Reply:
As mentioned above need the admin id to fetch the device token of admin to send notification.